main.py

```diff
@@ -178,13 +180,23 @@ def start_cron():
 
 def custom_data(post):
     return post.date_local >= datetime(2020, 3, 15)
+
+def zip_non_used_files():
+    for hashtag in HASHTAG_LABELS:
+        path = "./#" + hashtag
+        arqZip = zipfile.ZipFile("#" + hashtag + "ZIP.zip", "w", zipfile.ZIP_DEFLATED)
+        for file in os.listdir(path):
+            arqZip.write(os.path.join(path, str(file)), str(file))
```

Comment:
Why do you need to cast `file` to string? 🤔

Reply:
💅  If you indeed need this cast, I would recommend to create a variable in order to make the code more clear and uncluttered.

```suggestion
            file_name = str(file)
            arqZip.write(os.path.join(path, file_name), file_name)
```

Reply:
Why do you require the second argument in `arqZip.write`? I feel like you are passing the file name twice 🤔